data.sql

```diff
@@ -14,3 +14,70 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Boarmon', date '2005-06-07', 7, true, 20.4);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Blossom', date '1998-10-13', 3, true, 17);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Ditto', date '2022-05-14', 4, true, 22);
+
+/* Insert data into owners table */
+INSERT INTO owners(full_name, age)
+VALUES 
+    ('Sam Smith', 34),
+    ('Jennifer Orwell', 19),
+    ('Bob', 45),
+    ('Melody Pond', 77),
+    ('Dean Winchester',14 ),
+    ('Jodie Whittaker', 38);
+
+/* Insert data into species table */
+INSERT INTO species (name)
+VALUES
+    ('Pokemon'),
+    ('Digimon');
+
+/* Modify animals' species_id column */
+UPDATE animals 
+SET species_id = 2 
+WHERE name LIKE '%mon';
+
+UPDATE animals 
+SET species_id = 1  
+WHERE species_id IS NULL;
+
+/* Modify animals' owners_id column */
+
+UPDATE animals 
+SET owner_id = 1 
+WHERE name='Agumon';
+
+UPDATE animals 
+SET owner_id = 2 
+WHERE name='Gabumon';
+
+UPDATE animals 
+SET owner_id = 2 
+WHERE name='Pikachu';
+
+UPDATE animals 
+SET owner_id = 3 
+WHERE name='Devimon';
```

Comment:
- [OPTIONAL] Great job here 👏🏻 However you can use this syntax as well which is better than the one you used so far ⬇️ 

   ```sql
   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith')
   WHERE name = 'Agumon';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Jennifer Orwell')
   WHERE name = 'Gabumon' OR name = 'Pikachu';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Bob')
   WHERE name = 'Devimon' OR name = 'Plantmon';
       ...
```

Reply:
Hello @BenMukebo, thank you for taking out time to go through this PR and giving this optional change, I have gone ahead to implement the change in my code too.